PlaceRouteHierFlow/placer/Placer.cpp

```diff
@@ -268,6 +268,9 @@ std::map<double, std::pair<SeqPair, ILP_solver>> Placer::PlacementCoreAspectRati
   curr_sol.cost = curr_cost;
   oData[curr_cost] = std::make_pair(curr_sp, curr_sol);
   ReshapeSeqPairMap(oData, nodeSize);
+
```

Comment:
@854768750 I think this was supposed to be for debugging. This will skip the whole SA part.

Reply:
It was indeed. Let me revert that.

Reply:
It is merged from commits.

Reply:
Yes. I think it is for this testcase only